types/light.go

```diff
@@ -0,0 +1,222 @@
+package types
```

Comment:
Would it make sense for the light client to house the types that are only used in the light client? We have been following the convention of putting types in the `/types` but it may be good to consider types that are for specific modules/reactor to be housed in the module/reactor.

Reply:
yeah I just did this out of convention. I don't mind creating a separate types directory in the light client to house the light specific types. Depends what the others think?

Reply:
Yeah, I'm generally not a fan of these global `types` packages - personally I think packages should expose their own types directly.

Reply:
Would be nice to squeeze it in but I don't have a preference for now or later

Reply:
Later is fine with me